app/views/admin/admin_portals/index.html.erb

```diff
@@ -2,11 +2,11 @@
   <div class="tag-card crayons-card p-4 m:p-6 m:pt-4 mb-4 flex flex-row relative">
     <div class="mr-8">
       <div class="crayons-field__description">Last deployed</div>
-      <div><%= ApplicationConfig["RELEASE_FOOTPRINT"].presence || ENV["HEROKU_RELEASE_CREATED_AT"].presence || "Not Available" %></div>
+      <div><%= ForemInstance.deployed_at || "Not Available" %></div>
     </div>
     <div>
       <div class="crayons-field__description">Latest Commit ID</div>
-      <div><%= ApplicationConfig["FOREM_BUILD_SHA"].presence || ENV["HEROKU_SLUG_COMMIT"].presence || "Not Available" %></div>
+      <div><%= ForemInstance.latest_commit_id || "Not Available" %></div>
```

Comment:
Change the display for the admin portal page to reuse the ForemInstance methos instance of explicitly referencing the variables